ci/build/artefact-signing-key-sealed-secret.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: artefact-signing-key
+  namespace: verify-eidas-proxy-node-build
```

For master this would be correct but on release/single-country-legacy I think this would be:
```suggestion
  namespace: verify-proxy-node-build
```

---

Updated.